src/Enums/CalendarKeys.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @author Nicolas CARPi <nico-git@deltablot.email>
+ * @author Marcel Bolten <github@marcelbolten.de>
+ * @copyright 2024 Nicolas CARPi
+ * @see https://www.elabftw.net Official website
+ * @license AGPL-3.0
+ * @package elabftw
+ */
+
+declare(strict_types=1);
+
+namespace Elabftw\Enums;
+
+use Elabftw\Traits\EnumsTrait;
+
+enum CalendarKeys: string
+{
+    use EnumsTrait;
+
+    case AllEvents = 'all_events';
+    case Categories = 'categories';
+    case Items = 'items';
+    case Title = 'title';
+    case Todo = 'todo';
+    case UnfinishedStepsScope = 'unfinished_steps_scope';
+
+    public static function toArray(): array
```

Comment:
this would  fit very well in EnumsTrait.php

Reply:
Moved it to EnumsTrait